src/main/webapp/app/admin/report-management/report-management-logs.html

```diff
@@ -54,12 +56,26 @@
                         <td>{{log.dashboardName  }}</td>
                         <td>{{log.viewName}}</td>
                         <td>{{log.descripition}}</td>
-                        <td>{{log.channel}}</td>
-                        <td>{{log.thresholdMet?'Yes':'No'}}</td>
+                        <td ng-if="vm.reportType==='true'">{{log.thresholdMet?'Yes':'No'}}</td>
```

Comment:
reportType should boolean instead of string ...no need to use ' '. can change it in notification and here